### PR TITLE
Restyle windows and status bar

### PR DESCRIPTION
**Pull request by joshua-s**
This implements the new window styling that we discussed, minus the `theme-color` bits (which will come in #4).

### CODE REVIEW

File: laskyawm.css [hunk 1]
```diff
@@ -28,14 +28,15 @@ body {
 	padding: 0 5px;
 }
 .window.focused {
-	background-color: #D4E4EC;
+	box-shadow: 0px 0px 20px rgba(255, 255, 255, 0.3);
```
Comment (twiss) [inline laskyawm.css]: Could you make this black, at least for now?

File: laskyawm-bar.css [hunk 2]
```diff
@@ -1,12 +1,12 @@
 #bar {
-	background-color: rgba(0, 0, 0, .25);
+	background-color: rgba(255, 255, 255, 0.6);
+	color: #000;
```
Comment (twiss) [inline laskyawm-bar.css]: I know I said make windows the same as the bar, but minimized windows are a little hard to distinguish from the bar now. Maybe give minimized windows a more opaque color and the bar a different color?

File: laskyawm.css [hunk 3]
```diff
@@ -145,6 +138,19 @@ body {
...
+	position: fixed;
+	top: 0;
+	left: 100px;
+	width: 100%;
+}
+.window.maximized.maximized-max .tabs {
+	height: 100%;
+}
+.window.maximized.maximized-max iframe {
+	border-radius: 0;
```
Comment (twiss) [inline laskyawm.css]: I agree the maximized window thing is cool, but it's on top of the minimized windows now. Maybe hold off on this for now?

Comment (joshua-s) [inline laskyawm.css]: Aye aye!

File: laskyawm-bar.css (hunk 2 above)
Comment (joshua-s) [inline laskyawm-bar.css]: How's it now?

**Comment by twiss** [comment]
Merged as 81c98b3. Thanks!

**Pull request closed without merging**